deploy/roles/stata/templates/sysprofile.do.j2

```diff
@@ -0,0 +1,8 @@
+
+ set         odbcmgr         {{ stata_odbc_mgr }}
+ set         processors      {{ stata_processor }}
+ sysdir      set SITE        {{ stata_site }}
+
+    {% for printer in stata_printers %}
+        printer define {{ printer }}
+    {% endfor %}
```

Comment:
Please remove the indentation here so `printer...` is all the way to the left like the other settings.